src/System.Security.AccessControl/src/System/Security/AccessControl/SecurityDescriptor.cs

```diff
@@ -669,10 +669,8 @@ private static byte[] BinaryFormFromSddlForm(string sddlForm)
 nameof(sddlForm));
                     }
                     else if (error != Interop.Errors.ERROR_SUCCESS)
-                    {
-                        Debug.Assert(false, string.Format(CultureInfo.InvariantCulture, "Unexpected error out of Win32.ConvertStringSdToSd: {0}", error));
```

Comment:
The assert should remain, it's present in .NET Framework: http://referencesource.microsoft.com/#mscorlib/system/security/accesscontrol/securitydescriptor.cs,678

Reply:
done